server/seeds/test-users.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function (knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('users').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('users').insert([
+        {id: 88801, username: 'Tom_Hall', hash: , age:'8'},
+        {id: 88802, username: '', hash:, age: 5},
```

Comment:
I know it seems minor, but when you change the type of the **column** from string to integer, you need to change your **seed** data to match! So alter all the ages to ensure each one has an integer in it :smile:

Reply:
Actually never mind, you're fixing this in the other ticket. Approved!